pkg/sdk/testint/procedures_integration_test.go

```diff
@@ -555,3 +556,410 @@ func TestInt_OtherProcedureFunctions(t *testing.T) {
 		require.NoError(t, err)
 	})
 }
+
+func TestInt_CallProcedure(t *testing.T) {
```

Comment:
We have almost the same setups for the procedures of every type. Shouldn't we just run call after creation in the tests above (to prevent setup dulications)?

Reply:
i am not sure it is worth the trouble. the creation tests are already complicated enough, no need to put the call code in there as well.

Reply:
It would allow us to remove the duplicated code also and in this case I think it could be worth it. Especially because the use case we will have from Terraform side would be probable create (in resource) + call (in datasource), not createAndCall (from just one object).

But I don't want to block this change because there are other, more important topics, and this is mostly cosmetics. So closing.